### ISSUE TITLE
Changes from Meeting

### ISSUE THREAD

**Issue by kelseyfriedman** [JuliaDickey/Red-Sweater-Project-Rails#80]
- [x] Add Ada, Julia, Ashley and Jordan, Kelsey as admin 
- [x] VOLUNTEER: Add logos instead of bullet list in Volunteer Groups section. (ASHLEY sending logos to Julia.) 
- [x] Add partner logos to Our Team Section down below “Partners” (ASHLEY send logos to Julia)

- [x] Make link blue a little more easy to see (see FAQ page). Still blue, little more contrast next to grey
- [x] Move sponsor button behind Get Involved menu on main nav

- [ ] Pesky white bar above footer (fix top margin)

- [x] Let Jordan know there’s potential to add names to newsletter signup down the road  

- [x] Add soccer photo to Our Story (Ashley sent this. Use it to break up Our Story intro)

- [x] Changed “why do I do this work” to “Why do this work” on Our Story page quote.  DONE

- [x] Update Ada bio photo

- [x] While you're at it, Kelsey's is distorted. Does it need to get cropped to the right dimension before you upload it? 

- [x] Bios: Ambassadors and doctors need to have their title added back in. Ask Sona to review. (JULIA/KELSEY)

- [x] Add Ada and Ashley credentials and Lindsay. Ada will update her bio copy. ADA

- [x] Add more content into Julia’s bio

- [x] Confirm: Keeping monthly investment option for causes under $200 DONE

- [x] Break Tim’s profile into two paragraphs

- [x] link to new source to replace Malala link

- [x] Add link to cite 82% (Ashley sent two options in her email, KELSEY let me know which one you'd like to run with)

- [x] _X_ADA: update Engineers without borders link on The School page DONE

- [x] Add how a classroom changes lives video on the homepage after intro text before It Starts With a Sweater. 

- [x] Add all testimonials from old site 

- [x] SEE EMAIL: I DON'T THINK WE SHOULD REMOVE THE CTA. Bottom of volunteer page, instead of Donor CTA link to testimonials and anchor to testimonials section. 
- [x] _X_Volunteer map add a title: A Global Team. DONE

- [x] _X_Volunteer CTA updated to “help out from home” instead of “help out from your home country” DONE

- [x] _Add CTA to email at red sweater pros ….on Get Creative page. Content coming from Ashley to Julia. JULIA: adding into page and adding CTA

- [x] Sponsor page: “Safe” water? ADA thinking about how to correctly phrase this

- [x] new photo of Faustian without school uniform on KELSEY

- [x] Donate should be info@redsweaterproject.com globally.

- [ ] Add link to donation form and change request form to donate page ADA + JULIA

- [x] make sure all email links go to info, volunteer or tours, customize subject lines for info. 

- [x] Add older video to timeline in after Our Story. breaks up the page.

- [x] Globally remove “middle” school, just high school reference

**Comment by JuliaDickey** [comment]
Should the logos link to each company's site? I asked Ashley this, but no response so far. I have links in place if we need to add them.

**Comment by JuliaDickey** [comment]
It seems like "SPONSORS" would be more appropriate than "PARTNERS" for the logos section on The Team page. I don't think Nike would be happy to be listed as a partner, when what they provided was equipment. Not a big deal, but worth considering.

**Comment by kelseyfriedman** [comment]
@JuliaDickey I think it's nice to link the logos. That's pretty normal behavior. 
I went and grabbed them for you
http://www.awboregon.org/
http://courtsforkids.org/
https://embarkexplorationco.com/
http://www.ewb-usa.org/
http://www.gefoundation.com/
https://www.moondanceadventures.com/
https://www.pacificu.edu/future-graduate-professional/colleges/college-optometry
https://www.rotary.org/

I'm not sure what Unite is so you'll need to get the link from Ashley I think. 

I agree about the Partners vs Sponsors. Let me ask Ashley though.

**Comment by JuliaDickey** [comment]
Thank you! I've added these Volunteer links. Let me know about Sponsor vs Partner, and if that section should be linked as well.

**Comment by kelseyfriedman** [comment]
@JuliaDickey you may know this already but wanted to note that the video on the homepage doesn't resize responsively

**Comment by JuliaDickey** [comment]
Fixed the responsiveness of the video on the homepage.

**Comment by kelseyfriedman** [comment]
Hi Julia, I know you haven't passed this off to me yet but I saw that most of it was done so I went through this to get ahead of QA since I'm leaving. Here is some feedback - sorry if you are already planning to tackle any of this, I just didn't want to wait until Wednesday. 

Things Julia still needs
- [ ] Unite logo and URL form Ashley
- [ ] Decision about Partner vs Sponsor. I asked Ashley and she deferred to Jordan - haven't heard back. If i don't by your next meeting, please just ask Ashley to make the call. 

Other stuff do to
- [ ] Link logos on The Team page - Ashley doesn't seem to like linking off site... but I think we should. Here are the URLs 
https://timbersarmy.org/
http://www.benevity.com/
https://www.booksforafrica.org/
http://www.ge.com/
http://www.bushcenter.org/
infocus - not sure what this is as there are too many things when i google. ask ashley for more info?
https://www.maulfoster.com/
https://www.nationalcharityleague.org/
http://oregonpublichouse.com/
http://www.skytelafrica.com/
http://www.strongharvest.org/
http://tapasafari.co.tz/
http://www.thearushahotel.com/
http://www.uc.edu/

- [ ] Add ", Volunteer" after my job title to make it match the others
- [ ] On new testimonials there is a weird ampersand showing up —Rick &s; Jeri K., Agriculture Program Trainers
- [ ] If we use the photo of Glory and the volunteer for the new CTA, let's put something else in for the Blog CTA. There are lots of photos to choose from in the Dropbox or you could take one directly from a blog post (let me know if you want me to send you one). 
- [ ] For the new CTA, can we jazz up the language a little and add a period after the sentence. I don't want to rock the boat if Ashley wrote this, but I think it could be a little more interesting. How about: 
WHAT'S IT LIKE TO VOLUNTEER? Find out first-hand from past volunteers. READ TESTIMONIALS. 

Note for kelsey: Things that need to get QAd still when finished:
white bar footer
bios - titles
sponsor page- safe water
Add link to donation form and change request form to donate page ADA + JULIA

**Comment by JuliaDickey** [comment]
- [ ] Ask Ashley about which link to use for InFocus

**Comment by JuliaDickey** [comment]
I pulled an image from Dropbox for the Blog CTA.

**Comment by JuliaDickey** [comment]
Used "Ambassador" + last name, to refer to all 3 ambassadors on The Team page.

**Comment by JuliaDickey** [comment]
The verdict is "PARTNER" for the header above the logos on 'The Team' page. Here's Jordan's logic:
_So I'm torn on this because I don't like the use of the word "sponsor" in the sense that we use it for students. 

I don't want that to get confusing explicitly. Though I could see partnering be too strong, even just providing us gear in many ways is partnering. I'd lean towards partner personally...unless there has been negative feedback from someone that we've listed on that page that feels like that classification is explicitly over the line._

**Comment by kelseyfriedman** [comment]
Got it.

**Comment by kelseyfriedman** [comment]
Everything seems to be in order except two things below and the footer as you mentioned. So in the end, the only change to bios Ashley really needed/wanted was to call the 3 peeps Ambassador? I thought it was way more than that! She definitely changes her mind a lot :)

- [ ] Timbers Army link is broken (the url is in there twice)
- [ ] Faye Pitcher needs , Volunteer after her title too (I swear it was there before!)

**Comment by JuliaDickey** [comment]
She wanted to doctors to be called Dr so-and-so, but there are no doctors or doctorate peeps on any of the boards or the staff. I'll check with her tonight. 

Faye now has "Volunteer" added to the end of her title
Fixed the Timbers link. 

Hooray!

**Comment by kelseyfriedman** [comment]
All looks good. I'm adding your notes from the second meeting so we can track them and I added a new bug for Ashley's Stripe fixes. 

- [ ] Add Courts for Kids logo to Partners
- [ ] Add Occidental and Willamette University to Partners and volunteer logo sections. **(really, both pages? I wonder why)**
- [ ] Take out left over Ms. references in bios
- [ ] Reach out about nonprofit rates on stripe. Check rates on taking payments vs through web site.
- [ ] Find out: What’s max amount we can run through Stripe manually on the dashboard?
- [ ] Add Ada and Jordan as administrators on Stripe


Notes from Me:
- [ ] On volunteer page, the third row of logos doesn't seem to be aligned in the grid the first two are in
- [ ] In Ashley's bio you refer to her as "Holmer" just one time and I think it should be Ashley

**Comment by JuliaDickey** [comment]
Good catch on the logos. The above items are updated. Ashley wanted the universities on both pages because they have volunteered, but also partnered in a larger capacity. 

I emailed the Stripe nonprofit details to Ashley and we're setting up our nonprofit status now. 👍
